From 3fda434cfc3d6913f70177699444e87183a6c860 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Tue, 4 Oct 2022 16:39:49 +0200
Subject: [PATCH 012/529] ARM: dts: stm32: fix extra spaces around vbus
 supplies in stm32mp157c-ed1

Remove extra spaces before the closing brace of vbus_otg node, before the
vbus_sw label, and before the closing brace of vbus_sw node.

Change-Id: I0fa70f13f6f85d1a0b956a9b4ea6731c64a4ef57
Fixes: 9c559b1565e6 ("ARM: dts: stm32: add stpmic1 support on stm32mp157c ed1 board")
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157c-ed1.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/arch/arm/boot/dts/stm32mp157c-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157c-ed1.dts
@@ -278,13 +278,13 @@
 			vbus_otg: pwr_sw1 {
 				regulator-name = "vbus_otg";
 				interrupts = <IT_OCP_OTG 0>;
-			 };
+			};
 
-			 vbus_sw: pwr_sw2 {
+			vbus_sw: pwr_sw2 {
 				regulator-name = "vbus_sw";
 				interrupts = <IT_OCP_SWOUT 0>;
 				regulator-active-discharge = <1>;
-			 };
+			};
 		};
 
 		onkey {
